packages/ts-config-webpack-plugin/src/TsConfigWebpackPlugin.js

```diff
@@ -39,6 +40,36 @@ class TsConfigWebpackPlugin {
 		return path.resolve(__dirname, '../config/tsconfig.base.json');
 	}
 
+	/**
+	 * Check for known TSConfig issues and output warnings
+	 * @param {string} configFilePath
+	 */
+	verifiyTsConfig(configFilePath) {
+		const createConfigFileHost = {
+			onUnRecoverableConfigFileDiagnostic() {},
+			useCaseSensitiveFileNames: false,
+			readDirectory: typescript.sys.readDirectory,
+			fileExists: typescript.sys.fileExists,
+			readFile: typescript.sys.readFile,
+			getCurrentDirectory: typescript.sys.getCurrentDirectory,
+		};
+		const tsconfig = typescript.getParsedCommandLineOfConfigFile(configFilePath, {}, createConfigFileHost);
+		if (!tsconfig) {
+			console.warn('tsconfig could not be parsed');
```

Comment:
eventually output explicit as warning such as `Warning: tsconfig could could not be parsed`.

Reply:
Changed to: `console.warn(`Warning: could not parse "${configFilePath}".`);`